scheme/tekton.dev/v1alpha1_Pipeline.json

```diff
@@ -0,0 +1,2310 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
should we keep file name as v1alpha1? and how I can generate this JSON?

Reply:
As I say in description, I take schemas from redhat-developer/intellij-tekton#148
`intellij-tekton` has code to generate schemas.

Reply:
can we keep file name as v1beta1 instead of v1alpha1? as we are supporting v1beta1 for most of the cases?

Reply:
we have 2 schemes one for v1alpha1 another for  v1beta1